generators/entity-server/files.js

```diff
@@ -30,10 +27,22 @@ const serverFiles = {
 
 const microFrontEndFiles = {
     server: [
-        // eslint-disable prettier/prettier
-        detailsWidgetFiles.dynamic,
-        detailsWidgetFiles.copy,
-        tableWidgetFiles,
+        widgetFiles,
+        // {
```

Comment:
Is this dead?

Reply:
This was for shared resources, but I removed those. Will remove